.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "stage": 2
+    "stage": 2
```

Comment:
I just considered good idea to use 4 spaces for JSON like files, what do you guys think?